Dockerfile

```diff
@@ -11,7 +11,9 @@ RUN  \
      git clone https://github.com/minio/minio && cd minio && \
      git checkout master && go install -v -ldflags "$(go run buildscripts/gen-ldflags.go)"
 
-FROM alpine:3.12
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
+
+ARG TARGETARCH
 
```

Comment:
Don't need this in this file because @nitisht `go build` in the previous one builds automatically for the required ARCH - also this file is meant for only amd64 so we don't need to add this anyways.